Bottom: adc309b640325cdfc40d1efd8a66f440e62c92ab
Top:    0b6425c4eb2400fcd4a766eb6e1d20493bc62966
Author: Alexey Lyashkov <shadow@Alexeys-MacBook-Pro.local>
Date:   2016-01-15 12:25:51 +0300

Refresh of cli.patch

---

diff --git a/ib-sock.c b/ib-sock.c
index fd1635f..fa47955 100644
--- a/ib-sock.c
+++ b/ib-sock.c
@@ -116,7 +116,7 @@ static int cm_handler(struct rdma_cm_id *cmid, struct rdma_cm_event *event)
 		sock_event_set(sock, POLLERR);
 	}
 	/* if we return an error, OFED treat it as critical error and kill an ID,
-	 * dobule destroy id will result a panic. Probably we may return it early sometimes
+	 * dobule destroy id will result a panic. LNet uses it way
 	 * to save a resources. */
 	return 0;
 }
